### DISCUSSION TITLE
Use with newsboat / W3M

### DISCUSSION THREAD

**Discussion by wwe9112** [eafer/rdrview#18] [Q&A]
Have you guys tried to use this with W3m and Newsboat? I have 2 machines, one with Arch the other with POPOS. I was curious how it worked out for you what the process would be? I'm still learning I apologize if this is the wrong place to ask.

**Comment by eafer** [comment]
> Have you guys tried to use this with W3m and Newsboat? I have 2 machines, one with Arch the other with POPOS. I was curious how it worked out for you what the process would be?

Have you tried the example from the man page:

  BROWSER='rdrview -B lynx' newsboat

Just replace lynx with w3m, and I think it should work (though I didn't test
it). You can then alias newsboat to that.

I usually put a script in BROWSER that lets me choose a regular browser if
needed.
